app/services/users.js

```diff
@@ -39,3 +44,25 @@ exports.signin = (user) => {
     throw errors.defaultError(err.message);
   });
 };
+
+exports.update = (user, newData) => {
```

Comment:
why are you exporting this? If you don't want to use it outside this service you can declare it as a const an use it whenever you want.